assets/blocks/course-outline/status-control/index.js

```diff
@@ -0,0 +1,53 @@
+import { SelectControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * An enum of Status constants.
+ */
+export const Statuses = {
+	NOT_STARTED: 'NOT_STARTED',
+	IN_PROGRESS: 'IN_PROGRESS',
+	COMPLETED: 'COMPLETED',
+};
+
+/**
+ * Labels for each of the statuses.
+ */
+export const StatusLabels = {
+	[ Statuses.NOT_STARTED ]: __( 'Not Started', 'sensei-lms' ),
+	[ Statuses.IN_PROGRESS ]: __( 'In Progress', 'sensei-lms' ),
+	[ Statuses.COMPLETED ]: __( 'Completed', 'sensei-lms' ),
+};
+
+/**
+ * A component which controls the status preview for a block. It contains a group of buttons and a description.
+ *
+ * @param {Object}   props                 Component props. Extras will be handed through to the `SelectControl` component.
+ * @param {Array}    props.includeStatuses The ordered Status constants to include.
+ * @param {number}   props.status          The index of the current status.
```

Comment:
I think it's a `string` too, correct?

Reply:
https://github.com/Automattic/sensei/pull/3643/commits/11ac9787f8d6331693bdef310e29e950e9d7f17c